planner/core/task.go

```diff
@@ -2093,7 +2097,7 @@ func (p *PhysicalStreamAgg) attach2Task(tasks ...task) task {
 	} else {
 		attachPlan2Task(p, t)
 	}
-	t.addCost(p.GetCost(inputRows, true))
+	t.addCost(final.GetCost(inputRows, true))
```

Comment:
When calculating agg-cost, different aggs have different `aggFuncFactors`(see `basePhysicalAgg.getAggFuncCostFactor`), so we have to use the real final-agg here to calculate the cost.